js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -114,19 +114,21 @@ export const LaunchAssetExecutionButton: React.FC<{
 
   return (
     <>
-      <Button
-        intent={intent}
-        onClick={onClick}
-        icon={
-          state.type === 'loading' ? (
-            <Spinner purpose="body-text" />
-          ) : (
-            <Icon name="materialization" />
-          )
-        }
-      >
-        {label}
-      </Button>
+      <Tooltip content="shift-click to open the config editor">
```

Comment:
`Shift+click to add configuration`? I'm not sure we refer to it as the "config editor" anywhere in the product, or just in code.